sdk/python/feast/repo_config.py

```diff
@@ -47,6 +47,7 @@
     "snowflake.engine": "feast.infra.materialization.snowflake_engine.SnowflakeMaterializationEngine",
     "lambda": "feast.infra.materialization.aws_lambda.lambda_engine.LambdaMaterializationEngine",
     "bytewax": "feast.infra.materialization.contrib.bytewax.bytewax_materialization_engine.BytewaxMaterializationEngine",
+    "k8s.engine": "feast.infra.materialization.kubernetes.kubernetes_materialization_engine.KubernetesMaterializationEngine",
```

Comment:
Can we call this simply `k8s`? please.. 😄

Reply:
haha been considering this too, just followed the spark.engine right below